modules/nvidia-gpu-gcp-adding-a-gpu-node.adoc

```diff
@@ -0,0 +1,322 @@
+                        ],
+                        "tags": [
+                            "myclustername-2pt9p-worker"
+                        ],
+                        "userDataSecret": {
+                            "name": "worker-user-data"
+                        },
+                        "zone": "us-central1-a"
+                    }
+                }
+            }
+        }
+    },
+    "status": {
+        "availableReplicas": 1,
+        "fullyLabeledReplicas": 1,
+        "observedGeneration": 1,
+        "readyReplicas": 1,
+        "replicas": 1
+    }
+}
+----
+
+. View the existing nodes, machines, and machine sets by running the following command. Note that each node is an instance of a machine definition with a specific GCP region and {product-title} role.
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                                                             STATUS     ROLES                  AGE     VERSION
+myclustername-2pt9p-master-0.c.openshift-qe.internal             Ready      control-plane,master   8h      v1.25.4+77bec7a
+myclustername-2pt9p-master-1.c.openshift-qe.internal             Ready      control-plane,master   8h      v1.25.4+77bec7a
+myclustername-2pt9p-master-2.c.openshift-qe.internal             Ready      control-plane,master   8h      v1.25.4+77bec7a
+myclustername-2pt9p-worker-a-mxtnz.c.openshift-qe.internal       Ready      worker                 8h      v1.25.4+77bec7a
+myclustername-2pt9p-worker-b-9pzzn.c.openshift-qe.internal       Ready      worker                 8h      v1.25.4+77bec7a
+myclustername-2pt9p-worker-c-6pbg6.c.openshift-qe.internal       Ready      worker                 8h      v1.25.4+77bec7a
+myclustername-2pt9p-worker-gpu-a-wxcr6.c.openshift-qe.internal   Ready      worker                 4h35m   v1.25.4+77bec7a
+----
+
+. View the machines and machine sets that exist in the `openshift-machine-api` namespace by running the following command. Each compute machine set is associated with a different availability zone within the GCP region. The installer automatically load balances compute machines across availability zones.
++
+[source,terminal]
+----
+$ oc get machinesets -n openshift-machine-api
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                               DESIRED   CURRENT   READY   AVAILABLE   AGE
+myclustername-2pt9p-worker-a       1         1         1       1           8h
+myclustername-2pt9p-worker-b       1         1         1       1           8h
+myclustername-2pt9p-worker-c       1         1                             8h
+myclustername-2pt9p-worker-f       0         0                             8h
+----
+
+. View the machines that exist in the `openshift-machine-api` namespace by running the following command. At this time, there is only one compute machine per machine set, though a compute machine set could be scaled to add a node in a particular region and zone.
```

Comment:
"there is only one compute machine per machine set" - is a little unclear, does it mean that at this time, you can only configure one compute machine per machine set? If so, it could be helpful to restructure; for example, "You can only configure one compute machine per set, although you can scale a compute machine set to add a node in a particular region and zone."

"could be scaled" - passive voice - see suggestion above, if it is accurate (if it's something the user can configure)

"at this time" - I think it is good to avoid this as it implies different future functionality, which may or may not happen